Misc/NEWS.d/next/Library/2020-01-11-22-09-29.bpo-31485.DSvjsY.rst

```diff
@@ -0,0 +1 @@
+https://github.com/python/cpython/pull/17954#
```

Comment:
Please add a short description, not a link.

There will be a link to the corresponded issue on the tracker in any case.